src/components/player/index.vue

```diff
@@ -626,13 +626,29 @@ export default {
 				player.play();
 				MUSIC_VISUALS.start();
 				this.$store.commit('playing', true);
+				const artists = this.currentArtists.reduce((out, artist) => {
```

Comment:
Why not `this. updateDiscordActivity()`?

Reply:
Whoops. Of course that's a lot better, will adjust.